python/tutorials/09-persistent-matmul.py

```diff
@@ -460,7 +460,8 @@ def matmul_kernel_tma_persistent(a_desc_ptr, b_desc_ptr, c_desc_ptr,  #
     tile_id_c = start_pid - NUM_SMS
     num_pid_in_group = GROUP_SIZE_M * num_pid_n
 
-    for tile_id in tl.range(start_pid, num_tiles, NUM_SMS, flatten=True):
+    # Enable warp specialization to leverage async warp scheduling in the GPU.
+    for tile_id in tl.range(start_pid, num_tiles, NUM_SMS, flatten=True, warp_specialize=True):
```

Comment:
what will this do for Hopper?

Reply:
Nothing. Should I leave a comment about that?